### ISSUE TITLE
Better support for derpibooru links

### ISSUE THREAD

**Issue by DerVerruckteFuchs** [ccrama/Slide#2352]
Slide loads derpibooru links like regular web pages, but the main image that is supposed to be seen doesn't load. Either fixing how slide loads the content of the page so the image loads, or loading it like an imgur link would make slide usable for viewing derpibooru links.

**Comment by ccrama** [comment]
Is it possible you have Pro and ads are blocked in your browser? If so, Slide could be blocking these images as "ads". Also possibly that site needs cookies enabled, which you can enable in the Website 3 dot menu. If neither of those options work, enabling Chrome Customtabs in Settings > Link Handling should use Chrome to render webpages and will hopefully display the pages correctly!

**Comment by bb010g** [comment]
The images themselves shouldn't be the problem. Direct derpicdn.net links work just fine. Loading still fails with cookies enabled. Is there a way to disable Pro's ad block?

Ninja edit: Using [this post](https://reddit.com/r/mylittlepony/comments/5y0sma/do_the_sparkle/) for testing. After looking into the [ad blocking code](https://github.com/ccrama/Slide/blob/6791ded6bcde8a96aed9386a49a1bcd41dcbbebe/app/src/main/java/me/ccrama/redditslide/util/AdBlocker.java) and inspecting the page in Firefox, here's what I've got. It looks like the main `derpicdn.net` script isn't being loaded (`<script src="//derpicdn.net/assets/application-….js" async="async"></script>`). AdBlocker wouldn't be hitting this, as it's already loading other stuff from derpicdn and not giving it nothing. I'm thinking it might be that the browser you're using doesn't support async script loading, but according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#Browser_compatibility), it should be supported in everything by now. [Lightning](https://github.com/anthonycr/Lightning-Browser), which is also based off of `android.webkit`, shows it just fine and (as far as I can tell) has never had an issue with it (nothing in the issues/pull requests and no code mentioning async JS).

**Comment by DerVerruckteFuchs** [comment]
I had already enabled cookies, which weren't working apparently. I use the F-Droid version with the fancy Pro plugins for video and whatnot from Google Play. Enabling Chrome CustomTabs works for loading the images in derpibooru links. Direct image links had always worked fine, loading the webpage needs CustomTabs for the images to load properly.

**Comment by Tensounder54** [comment]
Yeah this is still an issue. I think slide should load image links from this site inline just like imgur links. I can look into doing this.
